static/templates/settings/settings-save-discard-widget.handlebars

```diff
@@ -1,12 +1,22 @@
 {{#if is_admin }}
-<div class="input-group organization-submission subsection-changes-save">
-    <button type="button" id="org-submit-{{section_name}}" class="button rounded sea-green hide" data-status="save">
-        {{t 'Save' }}
-    </button>
-</div>
-<div class="input-group subsection-changes-discard">
-    <button type="button" id="org-discard-{{section_name}}" class="button rounded hide">
-        {{t 'Discard changes' }}
-    </button>
+<div class="save-button-controls hide">
+  <div class="input-group organization-submission subsection-changes-save">
+      <div class="icon-button button primary save-button" type="button" id="org-submit-{{section_name}}" data-status="save">
+        <span class="fa fa-spinner fa-spin icon-button-loading"></span>
+        <span class="fa fa-check icon-button-icon"></span>
+        <span class="icon-button-text">
+            {{t 'Save Changes' }}
+        </span>
+      </div>
+  </div>
+  <div class="input-group subsection-changes-discard">
+
+    <div class="icon-button button discard-button" type="button" id="org-discard-{{section_name}}">
+      <span class="fa fa-times icon-button-icon"></span>
+      <span class="icon-button-text">
+        {{t 'Discard' }}
+      </span>
+    </div>
+  </div>
```

Comment:
I think the linter probably complained about this (`tools/lint`) but we generally do 4-space indentation in HTML templates in this codebase.